docs/dashboards/about.md

```diff
@@ -81,8 +81,8 @@ Live mode restrictions do not apply to Dashboard. 
 * A Dashboard can have up to 100 queries.
 * Dashboard chart properties are not retained when viewed from the Search page.
 * Chart properties are not retained when a chart is added to a Dashboard from the Search page.
-* A warning message will be displayed if the number of records returned from the lookup table is more than 1440.
-
+* No dashboard query can return more than 1440 data points. If the dashboard panel has more than 1440 pixels in width, additional resolution in data will be down-sampled to get rendered on the screen.
```

Comment:
```suggestion
* Dashboard queries cannot return more than 1440 data points. If the dashboard panel is wider than 1440 pixels, additional resolution in data will be down-sampled in order to render on the screen.
```

Reply:
@JV0812 please change to `Dashboard queries cannot return more than 1440 data points.` The rest of the explanation is not necessary.

Reply:
@oshoghi , all set! Can you pls approve the PR?